src/installation.rst

```diff
@@ -126,6 +137,8 @@ Cylc is split into a number of packages providing different functionality:
    Provides the scheduler "kernel" of Cylc along with the command-line.
 `Cylc UI Server`_
    Provides the "Cylc Hub" and the browser-based "Cylc GUI".
+:ref:`Cylc Rose`
+   Provides support for :ref:`Rose` suite configurations in Cylc workflows.
```

Comment:
(Slight niggle, probably not worth following up)

The link for Cylc Rose gets an underline whereas the others don't

![image](https://user-images.githubusercontent.com/61982285/128337201-62ded676-44eb-4ece-a7ba-daebe3671409.png)

Reply:
Yep, that's because it is a `:ref:` rather than a hyperlink and the Sphinx RTD theme has decided that one should get an underline and not the other.

Nothing much we can do without hacking their CSS.